src/components/ledger.js

```diff
@@ -413,6 +419,7 @@ class LedgerRecoveryForm extends Component {
     builder.inputs.forEach((input, index) => {
       const isSegwit = !!unspents[index].witnessScript;
 
+      if (!isSegwit) signatures[index] += "01"; // add SIGHASH_ALL flag to non-segwit signatures
```

Comment:
Should probably use `Transaction.SIGHASH_ALL`, since we already have it in scope anyway (it's used on line 410).